aries/aries_vcx_core/src/wallet/indy/mod.rs

```diff
@@ -134,3 +133,26 @@ const WALLET_OPTIONS: &str =
 const SEARCH_OPTIONS: &str = r#"{"retrieveType": true, "retrieveValue": true, "retrieveTags": true, "retrieveRecords": true}"#;
 
 impl BaseWallet for IndySdkWallet {}
+
+#[cfg(test)]
+pub mod test {
```

Comment:
The convention (which even you *mostly* follow) is to put test code in module called `tests`.

Reply:
Updated.

Reply:
Updated.